pkgs/os-specific/linux/libnvme/default.nix

```diff
@@ -60,7 +60,8 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   # mocked ioctl conflicts with the musl one: https://github.com/NixOS/nixpkgs/pull/263768#issuecomment-1782877974
-  doCheck = !stdenv.hostPlatform.isMusl;
+  # tests/sysfs is failing, so disable completely.
+  doCheck = false;
```

Comment:
wouldn't it make more sense to just disable `tests/sysfs` specifically?
https://github.com/linux-nvme/libnvme/blob/ea5600c843080db45590b5997176cc13d7a290bb/test/meson.build#L102

Reply:
Yeah, I still was about to figure that out, if there is any possibility to do that. That's why I've marked the pull request as draft. Sorry for the confusion.

Anyway, do you have an idea on how to do that properly?

Reply:
what does applying a patch which just comments out the linked subdir block do?

Reply:
I thought about that as well. It probably does the job, but that way the patch needs to be always carried. That doesn't feel like a long-term solution.

Reply:
Does not sound that complex of a patch.

Reply:
it's like a one-line change, it could even be done with sed/substituteInPlace, probably

Reply:
I like the idea to use sed. Will so that later!

Reply:
I'ved opened up #309932 with newer updates and fix for the sysfs shebang, ptal there?